spec/20-http_request_header_format.md

```diff
@@ -258,8 +258,8 @@ A simple example of a `list` with two `list-member`s might look like:
 
 ``` abnf
 list  = list-member 0*31( OWS "," OWS list-member )
-list-member = key "=" value
-list-member = OWS
+list-member =  key "=" value
+list-member =/ OWS
```

Comment:
ABNF syntax requires `=/` to indicate `OR`

Reply:
Good catch. Did you check the other ABNFs to make sure we're not making the same mistake in other places?

Reply:
In published spec, there is another place with this issue. In the level-2, this is the only place after https://github.com/w3c/trace-context/pull/386